app/classes/Transvision/ShowResults.php

```diff
@@ -367,6 +371,7 @@ public static function resultsTable($search_results, $recherche, $locale1, $loca
                       <a class='bug_link' target='_blank' href='{$bz_link[1]}'>
                         &lt;report a bug&gt;
                       </a>
+                    <span class='clipboard' data-clipboard-target='#{$clipboard_target_string2}' alt='Copy to clipboard'>&#9986;</span>
```

Comment:
I confess I find the scissors icon a bit confusing, but there's not much of an alternative, unless we use an image with a clearer copy icon (like we're doing to [invert locales](https://transvision.mozfr.org/img/swap_icon_horiz_18x18.png))